lib/src/kbasesearchengine/main/NarrativeInfoDecorator.java

```diff
@@ -81,12 +86,17 @@ public SearchTypesOutput searchTypes(final SearchTypesInput params, final String
     public SearchObjectsOutput searchObjects(final SearchObjectsInput params, final String user)
             throws Exception {
         SearchObjectsOutput searchObjsOutput = searchInterface.searchObjects(params, user);
+
+        if(searchObjsOutput.getAdditionalProperties().get(removedGuids) == null && System.getProperty(removedGuidsEnv) != null){
```

Comment:
whitespace around if. Actually, there's lots of inconsistent whitespace - do a pass through the code & fixup